k8s-training/provider.tf

```diff
@@ -2,6 +2,7 @@ terraform {
   required_providers {
     nebius = {
       source = "terraform-provider-nebius.storage.ai.nebius.cloud/nebius/nebius"
+      version = "0.3.19"
```

Comment:
Are we locking provider version now?

Reply:
I can remove it, but last time I tried, there were still issues with .22